Solidner/Views/Onboarding/FoodStartDateView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  FoodStartDateView.swift
+//  Solidner
+//
+//  Created by 황지우2 on 2023/11/13.
+//
+
+import SwiftUI
+
+struct FoodStartDateView: View {
+    let datePickerTopPadding = 51.0
+    @State var solidStartDate = Date()
```

Comment:
```suggestion
    @State private var solidStartDate = Date()
```
p3;
@State 는 private 으로 선언해주세요!